shared/test/otSdkDriver_test.js

```diff
@@ -992,6 +992,23 @@ describe("loop.OTSdkDriver", function() {
             new sharedActions.MediaConnected({}));
         });
 
+        it("should dispatch a connectionStatus action if both streams are up", function() {
+          driver._publishedLocalStream = true;
+
+          session.trigger("streamCreated", { stream: fakeStream });
+
+          // Called twice due to the VideoDimensionsChanged above.
+          sinon.assert.called(dispatcher.dispatch);
+          sinon.assert.calledWithMatch(dispatcher.dispatch,
+            new sharedActions.ConnectionStatus({
+              event: "Session.subscribeCompleted",
+              state: "receiving",
+              connections: 0,
+              sendStreams: 0,
```

Comment:
I assume this is 0 because _publishedLocalStream is artificially set?  Probably worth a comment for readability.